tester_app/app/assets/javascripts/application.js

```diff
@@ -1,3 +1,16 @@
+
+// It is important that generated/vendor-bundle must be before bootstrap since it is
+// exposing jQuery and jQuery-ujs.
+// DO NOT REQUIRE jQuery or jQuery-ujs in this file!
+// DO NOT REQUIRE TREE!
+// Must be on top
+//= require react_on_rails
+
+//= require generated/vendor-bundle
+//= require generated/app-bundle
+
```

Comment:
Because we are prepending to the application.js file, it is easier to just put react_on_rails above turbolinks:

        data = <<-DATA.strip_heredoc
          // DO NOT REQUIRE jQuery or jQuery-ujs in this file!
          // DO NOT REQUIRE TREE!

          // CRITICAL that generated/vendor-bundle must be BEFORE bootstrap-sprockets and turbolinks
          // since it is exposing jQuery and jQuery-ujs
          //= require react_on_rails

          //= require generated/vendor-bundle
          //= require generated/app-bundle

          // bootstrap-sprockets depends on generated/vendor-bundle for jQuery.
          //= require bootstrap-sprockets
        DATA